### PR TITLE
chart pull request

### PR DESCRIPTION
**Pull request by andreipetrescu95**

### CODE REVIEW

**Comment by grrowl** [comment]
LGTM !

File: deploy.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  if [ "$BUILD_NR_BLUE" -gt "$BUILD_NR_GREEN" ]; then
+    LIVE_COLOUR="blue"
+  else
+    LIVE_COLOUR="green"
+  fi
+fi
+
+if [ $LIVE_COLOUR == "blue" ]; then
+  TAG_GREEN=$APP_VSN-$BUILD
+  TAG_BLUE=$(kubectl -n "$ENV" get deployment "$ENV"-"$APP_NAME"-green --template="{{ (index .spec.template.spec.containers 0).image }}" | cut -d':' -f2 | tr -d '\n')
```
Comment (grrowl) [inline deploy.sh]: wait should this be `get deployment "$ENV"-"$APP_NAME"-blue`?

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  TRAFFIC_BLUE=0
+else
+  TAG_BLUE=$APP_VSN-$BUILD
+  TAG_GREEN=$(kubectl -n "$ENV" get deployment "$ENV"-"$APP_NAME"-green --template="{{ (index .spec.template.spec.containers 0).image }}" | cut -d':' -f2 | tr -d '\n')
+  BUILD_NR_BLUE=$CI_PIPELINE_IID
+  TRAFFIC_GREEN=0
+  TRAFFIC_BLUE=100
+fi
+
+helm upgrade --install --wait "$ENV"-"$APP_NAME" "$CHART" -f "$VALUES" --set greenTraffic="$TRAFFIC_GREEN" --set blueTraffic="$TRAFFIC_BLUE" --set image.greenTag="$TAG_GREEN" --set image.blueTag="$TAG_GREEN" --set buildNrGreen="$BUILD_NR_GREEN" --set buildNrBlue="$BUILD_NR_BLUE" --namespace "$ENV"
```
Comment (grrowl) [inline deploy.sh]: and `blueTag=$TAG_BLUE` here

**Comment by andreipetrescu95** [comment]
Very good catch Tom, I have fixed the problems. Thanks!

**Comment by grrowl** [comment]
LGTM !

**Pull request merged**